plugins/LilyPad/LilyPad.rc

```diff
@@ -450,21 +450,21 @@ BEGIN
     CONTROL         "Raw input",IDC_KB_RAW,"Button",BS_AUTORADIOBUTTON,22,40,112,10
     CONTROL         "DirectInput",IDC_KB_DI,"Button",BS_AUTORADIOBUTTON,22,52,112,10
     GROUPBOX        "Game Device APIs",IDC_STATIC,16,70,191,62
-    CONTROL         "DirectInput",IDC_G_DI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,82,65,10
-    CONTROL         "XInput",IDC_G_XI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,94,125,10
+    CONTROL         "DirectInput (For older controllers. Disable it for DS4)",IDC_G_DI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,82,183,10
+    CONTROL         "XInput (For modern controllers)",IDC_G_XI,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,94,125,10
     CONTROL         "DualShock 3 native mode (Requires libusb)",IDC_G_DS3,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,106,155,10
     CONTROL         "Monitor when in background",IDC_BACKGROUND,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,22,118,107,10
     GROUPBOX        "Mouse API",IDC_STATIC,216,16,192,85
     CONTROL         "Windows messaging (Recommended)",IDC_M_WM,"Button",BS_AUTORADIOBUTTON | WS_GROUP,223,27,137,10
     CONTROL         "Raw input",IDC_M_RAW,"Button",BS_AUTORADIOBUTTON,223,39,112,10
     CONTROL         "DirectInput",IDC_M_DI,"Button",BS_AUTORADIOBUTTON,223,51,112,10
     CONTROL         "Disable",IDC_M_DISABLE,"Button",BS_AUTORADIOBUTTON,223,63,39,10
-    CONTROL         "Start without mouse focus",IDC_MOUSE_UNFOCUS,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,223,75,98,10
+    CONTROL         "Start with unfocused mouse",IDC_MOUSE_UNFOCUS,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,223,75,105,10
```

Comment:
Older was better.

Reply:
Fair, i was playing around the idea to make it more readable but i'd prefer the variable to be IDC_MOUSE_FOCUS and say Start with mouse focus and the option by default disabled but that's just nitpicking from my side.